### DISCUSSION TITLE
NextJS+Express: body is injected with a display:none CSS property

### DISCUSSION THREAD

**Discussion by randomprogramming** [vercel/next.js#24350] [Help] [answered]
Hello, I'm trying to implement express into my nextjs typescript project. I copied the server.ts file from the examples and I got this:
```
import express, { Request, Response } from "express";
import next from "next";

const port = parseInt(process.env.PORT, 10) || 3000;
const dev = process.env.NODE_ENV !== "production";
const app = next({ dev });
const handle = app.getRequestHandler();

app.prepare().then(() => {
  const server = express();

  server.all("*", (req: Request, res: Response) => {
    return handle(req, res);
  });

  server.listen(port, () => {
    console.log(`> Ready on http://localhost:${port}`);
  });
});
```

I used the superplate boilerplate to generate my NextJS project, so I have a custom _document and _app files, generated by the boilerplate. I run my project using `npx ts-node server.ts` and when I open `localhost:3000` in my browser, I get an empty page. When I open the element inspector, I can see that the body has a `display: none` rule. I found that the source is this tag:
![image](https://user-images.githubusercontent.com/48063901/115711041-55bdca00-a373-11eb-8399-1d5576f29d8a.png)

What am I doing wrong and why is my body getting hidden? Thanks

**Comment by apollonian** [comment, marked as answer]
This is a dev only style addition and should get removed unless something is wrong w/ the setup I believe. Are all the scripts being served properly? If possible do add a reproducible demo, pls! 

Update: These seem related: https://github.com/vercel/next.js/discussions/16104 https://github.com/vercel/next.js/issues/10285

**Reply by randomprogramming** [reply]
Yeah, there was something wrong with the setup. I just made a completely new project, copied all the files over and it seems to be working fine now, even though everything is pretty much the same in the new project lol...
